web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -75,6 +78,10 @@
     if (newAsset.id && !isSharedLink()) {
       const data = await getAssetInfo({ id: asset.id });
       people = data?.people || [];
+      await getFaces({ id: asset.id }).then((data) => {
+        faces = data;
+        hasUnassignedFaces = faces.length > people.length;
```

Comment:
This won't always work. The same person can appear multiple times in an image (mirrors, collages, etc.).

Reply:
Completely forgot about that. Changed it to compare to the amount of faces assigned to people.